lib/plugins/helper/feed_tag.js

```diff
@@ -1,9 +1,36 @@
 'use strict';
 
 function feedTagHelper(path, options = {}) {
-  const title = options.title || this.config.title;
+  const { config } = this;
+  const title = options.title || config.title;
+  const { feed } = config;
```

Comment:
If `feed` is only used when no `path` is given, it can be moved to L18.

Reply:
refactored